iceoryx_examples/ice_multi_publisher/README.md

```diff
@@ -1,5 +1,262 @@
-# Multiple publishers on the same topic
+# Multiple publishers for the same topic
+
+## Introduction
+A common use case is that often we have multiple sensors sending data of the same type, e.g. LIDAR data, and a subscriber is interested in the data of all those sensors. 
+
+This example shows how you can run multiple publishers and let them pblish of the same topic. 
+Here we only cover the specifics for this use case, for basic sending and receiving see the 
+ [icedelivery](../icedelivery/README.md) example. The example also demonstrates how you can change the maximum number of samples available at a subscriber when you resubscribe.
+## Run ice_multi_publisher
+
+Create four terminals and run one command in each of them. 
+
+    # If installed and available in PATH environment variable
+    iox-roudi
+    # If build from scratch with script in tools
+    $ICEORYX_ROOT/build/posh/iox-roudi
+
+    ./build/iceoryx_examples/ice_multi_publisher/iox-multi-publisher
+
+    ./build/iceoryx_examples/ice_multi_publisher/iox-subscriber
+
+    ./build/iceoryx_examples/ice_multi_publisher/iox-resubscriber
```

Comment:
I'd replace this section with some nice [asciinema](https://asciinema.org/) recordnings. This applies to all examples. Will record them once 0.9 went live.